### PR TITLE
LPS-136867

### PR DESCRIPTION
**Pull request by DiegoDOC**
Related Issue: https://issues.liferay.com/browse/LPS-136867

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/FieldBase.scss [hunk 1]
```diff
@@ -9,3 +9,9 @@
 		font-size: 0.875rem;
 	}
 }
+.lfr-ddm-legend {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/FieldBase.scss:12]: @DiegoDOC, can you please add a line break before the class `lfr-ddm-legend`? Usually, this is how we separate CSS classes to make code easy to read. :wink: 

Additionally, can you please send a new pull with just a single commit?

Thanks.

**Pull request closed without merging**